htdocs/fourn/commande/list.php

```diff
@@ -737,8 +743,29 @@
 		print $form->selectyesno('validate_invoices', 1, 1);
 		print '</td>';
 		print '</tr>';
+		print '<tr class="fieldrequired" style="display:none">';
+		print '<td>';
+		print $langs->trans('RefSupplier');
+		print '</td>';
+		print '<td>';
+		print '<input name="refsupplier" type="text" id="refsupplier">';
```

Comment:
when using a mass action, if you select a lot of supplier orders from different vendors, when we will create invoices, we will have 1 supplier_ref for each different vendor. So how can we enter here the value into a single field ?
Asking the field may be interesting if we select "one bill by third" (you manage this, good), but also if the supplier is the same for all selected purchase order.